### PR TITLE
Add "Send CORS Preflights for Private Network Access"

### PR DESCRIPTION
**Pull request by ericlaw1979**
**Rendered article for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/web-platform/site-impacting-changes?branch=pr-en-us-1727

PTAL @mikehoffms

### CODE REVIEW

File: microsoft-edge/web-platform/site-impacting-changes.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ Review this article often.  The Microsoft Edge team updates this article as thin
...
+| Send CORS Preflights for Private Network Access | v98 | | Send a CORS [preflight](https://developer.chrome.com/blog/private-network-access-preflight/) request before a page from the internet may request resources from a local network (intranet). The intranet server should respond to the preflight with explicit permission to access the resource. This result of this check is not yet enforced. This change is happening in the Chromium project, on which Microsoft Edge is based. For more information, see the [Chrome Platform Status entry](https://chromestatus.com/feature/5737414355058688). Two compatibility policies are available to suppress the CORS preflight request: [InsecurePrivateNetworkRequestAllowed](/deployedge/microsoft-edge-policies#insecureprivatenetworkrequestsallowed) and [InsecurePrivateNetworkRequestAllowedForUrls](/deployedge/microsoft-edge-policies#insecureprivatenetworkrequestsallowedforurls). |
```
Comment (mikehoffms) [inline microsoft-edge/web-platform/site-impacting-changes.md:41]:
```suggestion
| Send CORS [preflight](https://developer.chrome.com/blog/private-network-access-preflight/) request for private network access | v98 | | Starting with v98, Microsoft Edge sends a CORS preflight request before a page from the internet is permitted to request resources from a local network (such as an intranet).  The intranet server should respond to the preflight request by providing explicit permission to access the resource.  This result of this check is not yet enforced.  This change is happening in the Chromium project, on which Microsoft Edge is based.  For more information, see the [Chrome Platform Status entry](https://chromestatus.com/feature/5737414355058688).  Two compatibility policies are available to suppress the CORS preflight request: [InsecurePrivateNetworkRequestAllowed](/deployedge/microsoft-edge-policies.md#insecureprivatenetworkrequestsallowed) and [InsecurePrivateNetworkRequestAllowedForUrls](/deployedge/microsoft-edge-policies.md#insecureprivatenetworkrequestsallowedforurls). |
```

Comment (mikehoffms) [inline microsoft-edge/web-platform/site-impacting-changes.md:41]:
```suggestion
| Send CORS Preflights for Private Network Access | v98 | | Send CORS [preflight](https://developer.chrome.com/blog/private-network-access-preflight/) request for private network access | v98 | | Starting with v98, Microsoft Edge sends a CORS preflight request before a page from the internet is permitted to request resources from a local network (such as an intranet).  The intranet server should respond to the preflight request by providing explicit permission to access the resource.  This result of this check is not yet enforced.  This change is happening in the Chromium project, on which Microsoft Edge is based.  For more information, see the [Chrome Platform Status entry](https://chromestatus.com/feature/5737414355058688).  Two compatibility policies are available to suppress the CORS preflight request: [InsecurePrivateNetworkRequestAllowed](/deployedge/microsoft-edge-policies.md#insecureprivatenetworkrequestsallowed) and [InsecurePrivateNetworkRequestAllowedForUrls](/deployedge/microsoft-edge-policies.md#insecureprivatenetworkrequestsallowedforurls). |
```

Comment (mikehoffms) [inline microsoft-edge/web-platform/site-impacting-changes.md:41]:
```suggestion
| Send CORS preflight requests for Private Network Access | v98 | | Starting with v98, Microsoft Edge sends a CORS [preflight](https://developer.chrome.com/blog/private-network-access-preflight/) request before a page from the internet is permitted to request resources from a local network (such as an intranet).  The intranet server should respond to the preflight by providing explicit permission to access the resource.  This result of this check is not yet enforced.  This change is happening in the Chromium project, on which Microsoft Edge is based.  For more information, see the [Chrome Platform Status entry](https://chromestatus.com/feature/5737414355058688).  Two compatibility policies are available to suppress the CORS preflight request: [InsecurePrivateNetworkRequestAllowed](/deployedge/microsoft-edge-policies.md#insecureprivatenetworkrequestsallowed) and [InsecurePrivateNetworkRequestAllowedForUrls](/deployedge/microsoft-edge-policies.md#insecureprivatenetworkrequestsallowedforurls). |
```

**Review by mikehoffms — CHANGES_REQUESTED** [review]
My comment (Suggestion from the writer/editor pass) is ready for you to selectively incorp.

**Comment by ericlaw1979** [comment]
I _think_ I've updated this PR with the changes you've requested.

**Review by mikehoffms — APPROVED** [review]

**Review by mikehoffms — APPROVED** [review]

**Pull request merged**